bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -313,7 +446,75 @@
 		<category>Number</category>
 		<state readOnly="true" pattern="%.0f %unit%"/>
 	</channel-type>
+	<channel-type id="energy">
+		<item-type>Number:Energy</item-type>
+		<label>@text/channelEnergyLabel</label>
+		<description>@text/channelEnergyDescription</description>
+		<category>Number</category>
```

Comment:
I would suggest to not set the category "Number" as the icon is missing in the classic icon set.
Same for all other channel types.

Reply:
Yes, thanks